src/Services/AlignmentSentinelService.php

```
@@ -0,0 +1,283 @@
+                continue;
+            }
+            $days = (int) $now->diff($blockedSince)->days;
+            if ($days < $maxDays) {
+                continue;
+            }
+
+            $workItemId = isset($story['parent_hl_item_id']) ? (int) $story['parent_hl_item_id'] : null;
+
+            // Skip if a dependency_tripwire alert already exists for this story
+            $existing = $workItemId !== null
+                ? DriftAlert::findActiveByWorkItemId($this->db, $orgId, $workItemId)
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\DriftAlert' in method 'checkBlockedDuration'. (undefined)

(StaticAccess)